### PR TITLE
Add doc on home-assistant plugin

### PR DESCRIPTION
**Pull request by pvmil**
Add description on the use of the home-assistant plugin

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/doods.markdown [hunk 1]
```diff
@@ -11,8 +11,16 @@ ha_domain: doods
...
+The DOODS software needs to be running, before this integration can be used. Options to run the DOODS software:
+### Run as Home Assistant add-on
+This is the easiest way when you are running supervised Home Assistant and want to run the DOODS software on the same machine as Home Assistant
+1. Add snowzach repository (https://github.com/snowzach/hassio-addons) to the add-on store (see [add-on documentation](https://www.home-assistant.io/hassio/installing_third_party_addons/) for more information )
+2. Go to the add-on store, you should now see the new repository
+3. Install DOODS 
+4. Go to dashboard, select DOODS and start DOODS
+
+### Run as a Docker container
+https://hub.docker.com/r/snowzach/doods
```
Comment (frenck) [inline source/_integrations/doods.markdown]: We should not provide a manual, that is for the upstream project to document. Not us.

Please adjust this to link to the add-on and docker containers, preferably with just a link to their installation instructions.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/doods.markdown (hunk 1 above)
Comment (pvmil) [reply source/_integrations/doods.markdown]: Thanks for your feedback, I've reworked this and will submit pullrequest

**Review by pvmil — COMMENTED** [review]
Review feedback fixed, approval required, no action by me (yet) right?
(Sorry for being relatively new to github and way of working)

File: source/_integrations/doods.markdown [hunk 2]
```diff
@@ -11,8 +11,9 @@ ha_domain: doods
 The `doods` image processing integration allows you to detect and recognize objects in a camera image using [DOODS](https://github.com/snowzach/doods/). The state of the entity is the number of objects detected and recognized objects are listed in the `summary` attribute along with quantity. The `matches` attribute provides the confidence `score` for recognition and the bounding `box` of the object for each detection category.
 
 ## Setup
-
-You need to have DOODS running somewhere. It's easiest to run as a [Docker](https://hub.docker.com/r/snowzach/doods) container.
+The DOODS software needs to be running, before this integration can be used. Options to run the DOODS software:
+* Run as Home Assistant add-on, repository: https://github.com/snowzach/hassio-addons
+* Run as a Docker container, https://hub.docker.com/r/snowzach/doods
```
Comment (frenck) [inline source/_integrations/doods.markdown]:
```suggestion

The DOODS software needs to be running before this integration can be used. Options to run the DOODS software:

- Run as [Home Assistant add-on](https://github.com/snowzach/hassio-addons)
- Run as a [Docker container](https://hub.docker.com/r/snowzach/doods)
```

**Comment by homeassistant** [comment]
Hi @pvmil, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16161).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @pvmil 👍

**Pull request merged**